moveit_core/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(Boost REQUIRED)
+find_package(Boost REQUIRED system filesystem date_time thread iostreams)
+# find_package(Boost REQUIRED)
```

Comment:
Remove this comment you are looking for boost in the previous line

Reply:
why are we touching moveit_core here for starters? @anasarrak refer to https://github.com/AcutronicRobotics/moveit2/pull/23#issuecomment-471318953 and please keep things tidy in terms of PRs, submit one per each package, it'll make things easier for us upstream.